ci/pipelines/build-and-deploy.yml

```diff
@@ -158,9 +158,10 @@ jobs:
               cp ../whitelisted-emails/domains.yml data/domains.yml
               cp ../organisations-list/organisations.yml data/organisations.yml
               bundle exec middleman build
-              cp -r . ../bundled/
+              cp -r build ../staticsite
+              cp manifest.yml ../staticsite
```

Comment:
Since you are not copying data, have you checked that the domains and organisations are still being handled appropriately?

Reply:
The files in `data` are used during the previous build step (line 160) but aren't needed in the staticsite dir. This was also the case previously.